core/src/main/java/org/streamprocessor/core/utils/SchemaUtils.java

```diff
@@ -101,7 +123,8 @@ private static FieldType getBeamFieldType(ColumnSchema column) {
         }
 
         if ("STRUCT".equals(dcFieldType)) {
-            Schema structSchema = fromColumnsList(column.getSubcolumnsList());
+            List<ColumnSchema> subColumns = column.getSubcolumnsList();
+            Schema structSchema = fromColumnsList(subColumns);
```

Comment:
how is this different?

Reply:
Probably changed it for debugging, will change it back!